runtime/oti/arm64helpers.m4

```diff
@@ -76,6 +76,8 @@ define({JIT_GPR_SAVE_OFFSET},{eval(J9TR_cframe_jitGPRs+(($1)*ALen))})
 define({JIT_GPR_SAVE_SLOT},{[sp,{#}JIT_GPR_SAVE_OFFSET($1)]})
 define({JIT_FPR_SAVE_OFFSET},{eval(J9TR_cframe_jitFPRs+(($1)*8))})
 define({JIT_FPR_SAVE_SLOT},{[sp,{#}JIT_FPR_SAVE_OFFSET($1)]})
+define({JIT_VRF_SAVE_OFFSET},{eval(J9TR_cframe_jitFPRs+(($1)*16))})
+define({JIT_VRF_SAVE_SLOT},{[sp,{#}JIT_VRF_SAVE_OFFSET($1)]})
```

Comment:
Where do you use this `JIT_VRF_SAVE_SLOT`?

Reply:
Removed these unused definitions.